packer/scripts/windows/scoop-install-scoop.ps1

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
@peterzhuamazon can we rename this file to scoop-config?

Reply:
Can rename later.

Reply:
Renamed now.